core-bundle/src/Command/ResizeImagesCommand.php

```diff
@@ -97,169 +116,240 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             throw new \RuntimeException('Deferred resizer not available');
         }
 
+        if (!$output instanceof ConsoleOutputInterface) {
+            throw new \InvalidArgumentException(sprintf('Output must be an instance of "%s"', ConsoleOutputInterface::class));
+        }
+
+        $this->io = new SymfonyStyle($input, $output->section());
+
+        if (null !== $image = $input->getOption('image')) {
+            return $this->resizeImage($image);
+        }
+
         $timeLimit = (float) $input->getOption('time-limit');
         $throttle = (float) $input->getOption('throttle');
-        $concurrent = (int) $input->getOption('concurrent');
+        $concurrent = (float) $input->getOption('concurrent');
 
-        if ($timeLimit < 0) {
-            throw new InvalidArgumentException(sprintf('Time-limit value "%s" is invalid.', $timeLimit));
+        if (1.0 !== $throttle) {
```

Comment:
Should't we check `$input->hasOption()` instead?

Reply:
I don’t think so because `$input->hasOption()` would always return true because the option exists in the definition of the command.

Reply:
Ah, you are right. Then we should probably check `null === $input->getOption()`, because if someone uses the option with `1.0`, the deprecation warning will not be triggered. 🙈

Reply:
Done in 54d17c6a6726fe159c78c5907a085378561acd82